src/styles/modules/_contact.module.scss

```
@@ -6,4 +6,9 @@
   .send-message {
     margin: 0;
   }
+
+  button:hover {
+    filter: brightness(0.85);
+    transition: filter 0.5s;
+  }
```

Seria uma boa colocar o transition fora do hover pois aí eu tenho o efeito de fadeOut também. 
Do jeito que está agora o transition ele é meio que condicional ao hover.

```suggestion
    transition: filter 0.5s;
  button:hover {
    filter: brightness(0.85);
  }
```

---

Oh, sure! Don't tried this, I was thinking in a way to make the animation on cursos out of the element. Thanks!

---

Hi Erick, I tested the way that was suggested, but not woks as I thought. But I find a good alternative to do so:
```
box-shadow: none;
    transition: box-shadow 0.4s;
  }

  a:hover {
    box-shadow: 0 2px white;
    transition: box-shadow 0.8s;
  }
```